pkg/cache/server/server.go

```diff
@@ -52,9 +52,8 @@ type preparedServer struct {
 }
 
 func (s *Server) PrepareRun(ctx context.Context) (preparedServer, error) {
-	logger := klog.FromContext(ctx).WithValues("component", "cache-server")
 	if err := s.apiextensions.GenericAPIServer.AddPostStartHook("bootstrap-cache-server", func(hookContext genericapiserver.PostStartHookContext) error {
-		logger = logger.WithValues("postStartHook", "bootstrap-cache-server")
```

Comment:
or make it just `logger :=`?

Reply:
the context of the hook lacks potentially some keys.

Reply:
pushed, ptal.